runtime/stdlib/account.go

```diff
@@ -699,7 +699,7 @@ func newAccountKeysGetFunction(
 				var err error
 				var accountKey *AccountKey
 				errors.WrapPanic(func() {
-					accountKey, err = provider.GetAccountKey(address, index)
+					accountKey, err = provider.GetAccountKey(address, uint32(index))
```

Comment:
not sure how to do the cast here safely.

Reply:
Fixed in [63abfd5](https://github.com/onflow/cadence/pull/3465/commits/63abfd5ec7b0d11bf142eead83591bf2e6c83ebb)